skops/hub_utils/_hf_hub.py

```diff
@@ -320,6 +323,11 @@ def init(
             task=task,
             data=data,
         )
+
+        with open(model, "rb") as f:
+            model = load(f)
+            model_card = card.Card(model, metadata=card.metadata_from_config(dst))
+            model_card.save(dst / "README.md")
```

Comment:
thanks, could you please write a test and check if the file is created after `init`?

Reply:
Do you think it would be nice if there would be another test that makes sure card is overridden after user creates it manually? Or is it too much? 😅

Reply:
yeah that also makes sense.

Reply:
I added both tests. I am not experienced writing tests, so I am not sure if that's the way you would implement them. After testing each test individually, they both pass.